parent/pom.xml

```diff
@@ -113,6 +131,12 @@
                 <groupId>org.xmlunit</groupId>
                 <artifactId>xmlunit-core</artifactId>
                 <version>${xmlunit.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>jakarta.activation</groupId>
+                        <artifactId>jakarta.activation-api</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
Is this really necessary? It's just a test dependency.

Reply:
I saw it introduces a 1.x activation. It's better to exclude it to make it clean: 
```
[INFO] +- org.xmlunit:xmlunit-core:jar:2.8.2:test
[INFO] |  \- jakarta.xml.bind:jakarta.xml.bind-api:jar:2.3.3:test
[INFO] |     \- jakarta.activation:jakarta.activation-api:jar:1.2.2:test
```

Reply:
I see it introduces  activation 1.x api, so better to exclude this dependency .
```
[INFO] +- org.xmlunit:xmlunit-core:jar:2.8.2:test
[INFO] |  \- jakarta.xml.bind:jakarta.xml.bind-api:jar:2.3.3:test
[INFO] |     \- jakarta.activation:jakarta.activation-api:jar:1.2.2:test
```